architecture/core_objects/kubernetes_model.adoc

```diff
@@ -220,8 +220,8 @@ documentation].
 
 == Service
 
-A service provides functionality to a set of pods running inside of a Kubernetes
-cluster, and are determined by the defined policy (sometimes called a
+A service provides functionality to a set of link:#pod[pods] running inside of a
+Kubernetes cluster, and are determined by the defined policy (sometimes called a
```

Comment:
"and are" subject is confusing here. Adjust to be clear that the set of pods is what's being determined. Actually this whole thing seems like a verbose way to discuss a relatively simple concept... a service identifies a set of pods and proxies the connections it receives to them.